### PR TITLE
Adminhtml feat (AdminSection): build dummy Magento admin section

### PR DESCRIPTION
**Pull request by PatrickKubica**

### CODE REVIEW

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config>
+    <tabs>
+        <webappmanifest translate="label">
+            <label>Web-App Manifest Configuration</label>
```
Comment (andrewhowdencom) [inline src/app/code/community/Sitewards/Webappmanifest/etc/system.xml]: Maybe label this PWA? We could have several extensions under here at some point

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <sort_order>1000</sort_order>
+        </webappmanifest>
+    </tabs>
+    <sections>
+        <webappmanifest translate="label">
+            <label>Configure Sitewards WebappManifest</label>
```
Comment (andrewhowdencom) [inline src/app/code/community/Sitewards/Webappmanifest/etc/system.xml]: `s/Configure //` - We're in the configuration; the word `Configure` is redundant

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <show_in_website>1</show_in_website>
+            <show_in_store>1</show_in_store>
+            <groups>
+                <settings translate="label">
+                    <label>Configure Sitewards WebappManifest</label>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <base_setting translate="label">
```
Comment (andrewhowdencom) [inline src/app/code/community/Sitewards/Webappmanifest/etc/system.xml:32]: `base_setting`?

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
+                        </base_setting>
+                    </fields>
+                </settings>
+            </groups>
+        </webappmanifest>
+    </sections>
+</config>
```
Comment (andrewhowdencom) [inline src/app/code/community/Sitewards/Webappmanifest/etc/system.xml]: Should have an EOL

File: src/app/code/community/Sitewards/Webappmanifest/etc/adminhtml.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```
Comment (andrewhowdencom) [inline src/app/code/community/Sitewards/Webappmanifest/etc/adminhtml.xml:1]: No license header love?

**Review by andrewhowdencom — CHANGES_REQUESTED** [review]
As always decent replies are better than blind changes. <3

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml (hunk 1 above)
Comment (PatrickKubica) [reply src/app/code/community/Sitewards/Webappmanifest/etc/system.xml]: very true

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml (hunk 3 above)
Comment (PatrickKubica) [reply src/app/code/community/Sitewards/Webappmanifest/etc/system.xml:32]: yes, this task was about adding a dummy setting, so we can expand the adminhtml grid later.
So at this moment we do not even know what we wanna configure later -> this is just a placeholder

File: src/app/code/community/Sitewards/Webappmanifest/etc/system.xml (hunk 4 above)
Comment (PatrickKubica) [reply src/app/code/community/Sitewards/Webappmanifest/etc/system.xml]: oh, I actually added them, but they were removed....
Lemme check that

**Review by andrewhowdencom — APPROVED** [review]

**Pull request merged**